libraries/AP_IQ_Motor/AP_IQ_Motor.h

```diff
@@ -119,7 +118,7 @@ class AP_IQ_Motor : public AP_ESC_Telem_Backend
     uint8_t _first_telem_motor = 0;
     uint8_t _last_telem_motor = 0;
 
-    static const uint8_t kSubCtrlCoast                  =  2;
-    static const uint8_t kTypePropellerMotorControl     = 52;
-    static const uint8_t kBroadcastID                   = 63;
+    static const uint8_t _kSubCtrlCoast                  =  2;
```

Comment:
This is more of a question for @vertiq-luca but if you have insight on this @vertiq-jordan let me know, since this is from his initial implementation, but are these necessary? Are these repetitive of existing sub ID and type ID stuff that is already  in the library files? I guess I'm just reluctant to have the same info in the same place twice without a good reason

Reply:
@iq-fred They do exist in our other libraries, but we don't have all the client files brought in for this implementation. If you think that would be a better way to do it, that's fair enough.

Reply:
I won't belabor the point for now if you guys just want to pull in a subset of the library. If at some point we do pull in the full library, we should avoid the repetition.